src/app/components/autocomplete/autocomplete.component.html

```diff
@@ -0,0 +1,23 @@
+<mat-form-field appearance="outline">
+	<mat-label>npm package name</mat-label>
+	<input matInput [formControl]="addPackage" [matAutocomplete]="auto" />
+	<mat-hint>Ex: 'canjs'</mat-hint>
+	<mat-error appErrorHandler [errors$]="addPackage.errors | toObservable" [handleErrors]="packageErrorsHandler">
+	</mat-error>
+
+	<!-- display options -->
+	<mat-autocomplete #auto="matAutocomplete">
+		<mat-option *ngFor="let option of autocompleteOptions$ | async" [value]="option"> {{ option }} </mat-option>
+	</mat-autocomplete>
+</mat-form-field>
+
+<!-- add button -->
+<button
+	mat-raised-button
+	type="button"
+	color="primary"
+	[disabled]="!addPackage.value || addPackage.invalid"
```

Comment:
This disable logic once migrated back to `AppComponent` will likely not work once migrated back without the suggested change to `validate()` method